src/libcore/char/mod.rs

```diff
@@ -92,15 +92,15 @@ const MAX_THREE_B: u32 = 0x10000;
 /// [Unicode Scalar Value]: http://www.unicode.org/glossary/#unicode_scalar_value
 /// [Code Point]: http://www.unicode.org/glossary/#code_point
 #[stable(feature = "rust1", since = "1.0.0")]
-pub const MAX: char = '\u{10ffff}';
+pub const MAX: char = char::MAX;
```

Comment:
pub use instead

Reply:
Not possible because here `char::` is not a module

Reply:
rustdoc can export [primitive types](https://doc.rust-lang.org/nightly/src/core/primitive.rs.html) but not associated constants. This could be improved.
Anyway, it currently rustdoc can't do that, just ignore these comments.